docs/security.md

```diff
@@ -368,5 +368,12 @@ launch -Dapex.attr.SSL_CONFIG="{\"keyStorePath\":\"/opt/apexCli/ssl/myapex-keyst
 
 Apec CLI will copy the keystore file `/opt/apexCli/ssl/myapex-keystore.jks` to the destination STRAM node and also pass on the keystore password values to the STRAM. As mentioned above, you can also use configuration files to supply the value of `apex.attr.SSL_CONFIG`.
 
+#### Updating Trust-store for the App Proxy
+
+You need to ensure that all end-points connecting to the STRAM Web service trust the SSL certificate when SSL is enabled.
+This is especially true for the [Web Application Proxy](https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/WebApplicationProxy.html) which connects to the STRAM Web service whenever you access the service through the App Master Proxy HTTPS URL. 
+If you use a self-signed or untrusted certificate, you will need to add that certificate to the trust-store used by the RM Web Application Proxy as described [here](https://www.cloudera.com/documentation/enterprise/5-8-x/topics/cm_sg_create_key_trust.html#concept_u35_w2m_l4) and update `ssl-client.xml` to use the trust-store as described [here](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/SecureMode.html#Data_Encryption_on_HTTP)
```

Comment:
Were these instructions tested on a SSL enabled cluster? I don't see how it will work without the fix for [YARN-2554](https://issues.apache.org/jira/browse/YARN-2554).

Reply:
@vrozov yes, this was tested on an SSL enabled cluster. That was a Hadoop distro from MapR.

Reply:
@sanjaypujare Does MapR hadoop distro have YARN-2554 fixed? The instructions provided are not specific to MapR distribution and it does not look that they will work in a generic case.

Reply:
It should mention the YARN version that has the issue fixed as well as the JIRA, as vendors may include selective JIRAs into their distros.